docs/docs/APIGuide/Module.md

```diff
@@ -158,6 +158,8 @@ model.evaluate(val_rdd, batch_size, val_methods)
 
 Use `evaluate` on the model for evaluation. The parameter `dataset` (Scala) or `val_rdd` (Python) is the validation dataset, and `vMethods` (Scala) or `val_methods`(Python) is an array of ValidationMethods. Refer to [Metrics](Metrics.md) for the list of defined ValidationMethods.
 
+For most cnn models, you can set `bigdl.engineType` as **mkldnn** to get better performance when doing evaluation.
```

Comment:
cnn -> CNN

Reply:
For most CNN models, it's recommended to enable MKL-DNN acceleration by specifying `bigdl.engineType ` as 'mkldnn'

Reply:
good, changed.